src/MessageContainer.js

```diff
@@ -141,6 +141,7 @@ export default class MessageContainer extends React.Component {
         <ListView
           enableEmptySections={true}
           keyboardShouldPersistTaps={true}
+          keyboardDismissMode={this.props.keyboardDismissMode || 'interactive'}
```

Comment:
Perhaps you can use [`defaultProps`](https://facebook.github.io/react/docs/react-without-es6.html#declaring-prop-types-and-default-props) instead of this.

See [MessageContainer.js#L161](https://github.com/Osedea/react-native-gifted-messenger/blob/f335e7157d88a5cf6e87e3dbf9f9ef02e2c47c2e/src/MessageContainer.js#L161)